src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -237,10 +244,17 @@
 				contentDefinition.isSvg
 					? document.createElementNS("http://www.w3.org/2000/svg", contentDefinition.tagName)
 					: document.createElement(contentDefinition.tagName);
-			element.id = String(contentDefinition.id);
-			element.setAttribute("XamlType", contentDefinition.type);
-			element.setAttribute("XamlHandle", `${contentDefinition.handle}`);
-			if (contentDefinition.isFrameworkElement) {
+
+			element.id = contentDefinition.id;
+
+			const uiElementRegistration = this.uiElementRegistrations[this.handleToString(contentDefinition.uiElementRegistrationId)];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)